pages/dkp/konvoy/2.1/cli-tools/helm/index.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: layout.pug
+navigationTitle: Helm CLI
+title: Helm CLI
+menuWeight: 5
+excerpt: Helm CLI Usage
+beta: false
+enterprise: false
+---
+
+This section describes how to install the Helm CLI. Helm can be installed either from source or from pre-built binary releases. This topic focuses on installing the latest Helm binaries, which is known as Helm 3.
+
+You may encounter some situations where you need to use an older version of Helm, known as Helm 2. If so, see the Helm documentation.
+
+## What Is Helm
+
+Helm is a tool that streamlines installing and managing Kubernetes applications. It provides managed packages called _charts_, which is a collection of related Kubernetes resources. Helm allows for customization, installation, upgrades, and rollback of packages.
+
+### Key Features
+
+- Repository of managed packages
+- Custom repositories for personal packages
+- Mechanisms for upgrading packages
+- Templating
+
+## Before you begin
+
+You need certain software configurations and settings before you start this procedure. This procedure requires the following items and configurations:
+
+- A running Kubernetes cluster
+
+## Install Helm from the binaries
+
+Every Helm [release][helm-release] provides binaries for a variety of operating systems. These binary versions can be manually downloaded and installed.
+
+1.  Download and unpack the [latest][helm-release-latest] release. Here is an example for the version [3.3.4][helm-3.3.4].
```

Comment:
Needs the arm64->amd64 changes listed above

Reply:
I copied over the changes from the 2.0 page to 2.1.